src/platform/common/process/condaService.node.ts

```diff
@@ -181,20 +181,20 @@ export class CondaService {
      * Even if not, we'll update this with the latest information.
      */
     private async getCachedInformation(): Promise<{ version: SemVer; file: Uri } | undefined> {
-        const cachedInfo = this.globalState.get<{ version: string; file: Uri; fileHash: string } | undefined>(
+        const cachedInfo = this.globalState.get<{ version: string; file: string; fileHash: string } | undefined>(
```

Comment:
This doesn't look related to the logging change. Was this from the file => Uri changes? Is the issue here that the complex Uri type wasn't working in the cache?

Reply:
> This doesn't look related to the logging change.


Agreed, i've added that comment in the PR `Also fixed a bug where we were cashing a Uri & caused things to fall over ( as uri cannot be serialized into JSON)`

> s the issue here that the complex Uri type wasn't working in the cache?

Yes, Uri gets serialized into JSON when storing in cache and such a complex class cannot be serialized & deserialized.